### DISCUSSION TITLE
Same prompt different results in Vlad and Auto1111

### DISCUSSION THREAD

**Discussion by dtxn** [vladmandic/sdnext#631] [Q&A] [answered]
Hi, I use the same model, prompt and seed in Vlad and in Auto1111, but get totally different results. What could be a problem ?

**Comment by vladmandic** [comment]
model/prompt/seed are just tip of the iceberg, there are so many other things that impact end results.

**Reply by dtxn** [reply]
Maybe you have any idea what could be wrong ? Vlad was my primary app, but I had a lot of problems with Deforum, so today installed Auto1111 to check if here are the same problems. So I just used the same those options and got different results:
Model
SD VAE
Sampling method
Sampling size
Picture size
CFG Scale
Seed

Shortly- "From text" page is the same, I'm newbie, so very confused what I could miss.

**Comment by vladmandic** [comment]
best would to
- check if you're using same cuda/memory options (xformers, no-half, etc.)
- make sure that you don't have any 3rd part extensions that may change results in both repos
  some do without even making it obvious they are doing it because they are modifying how prompt is parsed
- generate image in a1111
- import image here and use send to text and run generate

**Reply by dtxn** [reply]
Thank you, I will check for the 3rd parties, but one thing I noticed, that when I load Vlad, so Torch 2.0 +cu118 is loaded, but when I load Auto1111, so only Torch 1.13.1 +cu117 is loaded. How could I fix that ?
![image](https://user-images.githubusercontent.com/132169795/235325539-b1c45ef8-1051-4058-b379-6f2ad39ef9a5.png)

**Reply by vladmandic** [reply]
thats not something to "fix", that is a core difference between a1111 and this repo.

**Reply by dtxn** [reply]
I got the same result as you suggested after importing image from Auto1111 to Vlad. So what it could mean, what I miss ? Something in settings ?

**Reply by dtxn** [reply]
> thats not something to "fix", that is a core difference between a1111 and this repo.

I mean why Torch 2.0 is not loading while in Vlad loading without any problems. I would like to load it the same in Auto and don't know how to do that.

**Reply by tkiikt** [reply]
See [this](https://github.com/vladmandic/automatic#optimizations). Vlad is optimized and A1111 is not.

**Reply by dtxn** [reply]
That I saw, but still I don't know what should I change in settings or where...

**Reply by vladmandic** [reply]
given large differences in underlying packages (e.g. torch), its never going to be *same*.
but if you set same cuda and cross-optimization settings and import image and use it as template to read params from, it should be really close. i don't know what else to tell you since i don't even know what differences you are seeing. at least upload both images - what's the point of asking to explain differences if we cannot see them?

**Comment by tkiikt** [comment, marked as answer]
if it helps: #334

**Reply by vladmandic** [reply]
yup, that thread sums it up nicely.

**Reply by dtxn** [reply]
Thank you, guys, I will check it closely.
